pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -151,6 +147,62 @@ async def run(self):
                 await slack_client.say(error_message, slack_thread)
             raise
 
+    async def _attach_builds(self):
+        """ attach the microshift builds to advisory
+        """
+        cmd = [
+            "elliott",
+            "--group", self.group,
+            "--assembly", self.assembly,
+            "--rpm", "microshift",
+            "find-builds",
+            "-k", "rpm",
+            "--member-only",
+            "--use-default-advisory", "microshift"
+        ]
+        ret, _, _ = await exectools.cmd_gather_async(cmd, check=False, env=self._elliott_env_vars)
```

Comment:
Any reason why `check=False` is used here?

Reply:
if check=True it will raise the error in async call and fail the job from there, the following step may not be executed.